crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -292,16 +267,9 @@ impl JsonRpcHandler {
         block_id: BlockId,
         request: Vec<BroadcastedTransactionWithType>,
     ) -> RpcResult<Vec<EstimateFeeOutput>> {
+        // TODO: move EstimateFeeOutput to types
```

Comment:
shouldn't this be moved in this PR?

Reply:
I decided to migrate step by step. This pr is focused mostly on transactions. Other stuff will me moved slowly in following prs.